Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs

```diff
@@ -494,17 +499,27 @@ public void DisplayCurrentImage()
         {
             if (!_destroyed)
             {
-                if (Data.DisplayAd)
+                //Debug.Log(gameObject.name + "Single? " + Single);
+                if (Data.DisplayAd && Single)
                 {
                     AdContainer.SetActive(true);
                     foreach (var p in ProductVisualRules)
                     {
                         p.Visual.SetActive(false);
                     }
-                    if(Data.Texture && (displayImage.sprite == null || Data.Texture != displayImage.sprite.texture)) AdImage.sprite = Data.Texture.ToSprite();
+                    if(Data.Texture && (displayImage.sprite == null || Data.Texture != displayImage.sprite.texture))
+                    {
+                        AdImage.sprite = Data.Texture.ToSprite();
+                    }
+                    else 
+                    {
+                        AdImage.sprite = displayImage.sprite.texture.ToSprite();
+                    }
```

Comment:
Does this ever happen? Where we are trying to load an Ad but it doesn't show?

Reply:
I forgot to remove it, it was to test something.